chat-android/src/main/java/com/ably/chat/Presence.kt

```diff
@@ -26,7 +31,7 @@ interface Presence : EmitsDiscontinuities {
      * @param {Ably.RealtimePresenceParams} params - Parameters that control how the presence set is retrieved.
      * @returns {Promise<PresenceMessage[]>} or upon failure, the promise will be rejected with an [[Ably.ErrorInfo]] object which explains the error.
      */
-    suspend fun get(params: List<Params>): List<PresenceMember>
+    suspend fun get(params: List<Param> = listOf()): List<PresenceMember>
```

Comment:
I think the integration test should have detected the issue, since current `presence.get` will return an empty list even when `presence sync` is in progress